### PR TITLE
Geojson polygon generation

### PR DESCRIPTION
**Pull request by jocelynj**

### CODE REVIEW

File: tools/utils.py [hunk 1]
```diff
@@ -51,3 +54,87 @@ def print_header(title = ""):
...
+    if x != 0 and y != 0 and z != 0:
+        params = "%f-%f-%f" % (x, y, z)
+    else:
+        params = "0"
+    pgcursor.execute("SELECT id FROM polygons WHERE id = %s AND params = %s", (rel_id, params))
+    if pgcursor.fetchone():
+        return True
+
+    if create:
+        if x != 0 and y != 0 and z != 0:
```
Comment (frodrigo) [inline tools/utils.py:82]:
```suggestion
        if x != 0 or y != 0 or z != 0:
```

Comment (frodrigo) [reply tools/utils.py:82]: Better to reverse the then/else

Comment (jocelynj) [reply tools/utils.py:82]: Do you mean like this?
```
if (x, y, z) == (0, 0, 0):
  create_polygon(pgcursor, rel_id)
else:
  simplify_polygon(pgcursor, rel_id, x, y, z)
```

Comment (frodrigo) [reply tools/utils.py:82]: Yes

**Review by frodrigo — APPROVED** [review]

**Pull request merged**